README.md

```diff
@@ -79,6 +86,84 @@ The cli runs the minimum viable components for testing a rollup against the
 Astria stack, allowing developers to confirm that their rollup interacts with
 Astria's apis correctly.
 
+## Running Astria
+
+The `astria-go` cli is a tool to run a minimum viable Astria stack for
```

Comment:
There are lots of areas where we say things like "Using the cli", "with the cli", "after the cli is installed."

I think all these things are obvious since we're in the docs for the cli, but it's also too vague because you're actually talking about a specific command here (`dev`), not just the cli. I think you should just drop these types of sentences.

Sort of tangential, but we talk about `dev init` in the installation section. The installation sections should probably only go as far as to call `version`, then in this section we can have a 
`## Initialization` and `## Run` sub sections. Then it's obvious what part of the cli you're talking about because it's under its specific section. Now you no longer need to keep saying things like "Using the cli..."; it will just be implied.